particle/data/particle2008.csv

```diff
@@ -0,0 +1,558 @@
+ID,Mass,MassUpper,MassLower,Width,WidthUpper,WidthLower,I,G,P,C,Anti,Rank,Status,Name,Quarks,Latex
```

Comment:
I would suggest the following for these generated files, as a header:
- Explain which code generated them.
- Specify the input file.

I think this is important.

Reply:
I don't think adding the lines to the file would be easy - to keep the nice and simple csv parser instead of Pandas, I don't think we can have non-csv data in a csv file. However, I can add a README.md in the same directory that explains the generation and upkeep of these files.

Reply:
That's for sure an alternative. But it seemed from me (https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html) that headers were possible.

Reply:
Yes, in Pandas. I'm not requiring Pandas, just the standard library csv parser.